cfme/fixtures/vporizer.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
```

Comment:
I think you should split this fixture into its own PR and set this PR blocked by it so that it may be reviewed separately.